Sender/wrapper.py

```diff
@@ -0,0 +1,23 @@
+#This file is used for Mocking->counting the occurence of called buit-in function[Wrapper]
+
+import random
+import time
+
+class wrapper:#Decorator
```

Comment:
By convention, class names in python are `PascalCase`. This class calls a function and tracks how many times it's called. Can name it more specific, like 'FuncCallCounter`.